tools/Google.Cloud.Tools.ProjectGenerator/Program.cs

```diff
@@ -46,7 +46,7 @@ public class Program
         private const string DefaultVersionValue = "default";
         private const string GrpcPackage = "Grpc.Core";
         private const string DefaultGaxVersion = "2.2.1";
-        private const string GrpcVersion = "1.7.1";
+        private const string GrpcVersion = "1.8.0";
```

Comment:
We need to make sure the next release of every package is at least a minor version release (not a patch release).
Do we have tooling that ensures this? Or do we just need to be careful?

Reply:
Any GA'd releases have explicit versions for everything anyway, which is why only a subset of projects are updated in the second commit. We need to be careful to upgrade those manually when we do a next minor update...

(We also need to look at GAX, and update the protobuf generator and dependency to 3.5.1 at the same time. We should probably chat about that.)